test/metabase/email_test.clj

```diff
@@ -57,20 +58,13 @@
             ;; This will block the calling thread (i.e. the test) waiting for the promise to be delivered. There is a
             ;; very high timeout (30 seconds) that we should never reach, but without it, if we do hit that scenario, it
             ;; should at least not hang forever in CI
-            promise-value (deref p 30000 ::timeout)]
+            promise-value (deref p (if config/is-dev? 3000 30000) ::timeout)]
```

Comment:
nit: wdyt about multiplying using seconds for these magic numbers, and just multiplying by 1000 in the code?

Reply:
right!!!